extensions/node-menu/src/node-menu.tsx

```diff
@@ -62,15 +67,15 @@ export function NodeMenu(props: NodeMenuProps) {
   };
 
   const cordon = () => {
-    sendToTerminal(`kubectl cordon ${nodeName}`);
+    sendToTerminal(`${kubectlPath} cordon ${nodeName}`);
```

Comment:
Is this going to work if `kubectlPath` has a space in it?

Reply:
The call to `inspect` above was supposed to catch any possible cases like that. And it does, but how it handles strings with both `'` and `"` in them (by using `) is wrong and I should fix it.